spec/units/peep_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'peep'
+require 'database_helpers'
+
+describe Peep do
+  describe '.all' do
+    it 'returns all peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      peep = Peep.create(message: 'Peep 1', date: '2021-06-25')
+      Peep.create(message: 'Peep 2', date: '2021-06-25')
+      Peep.create(message: 'Peep 3', date: '2021-06-25')
+      
+      peeps = Peep.all
+
+      expect(peeps.length).to eq 3
+      expect(peeps.first).to be_a Peep
+      expect(peeps.first.id).to eq peep.id
+      expect(peeps.first.message).to eq 'Peep 1'
```

Comment:
Related to my previous comment, I'd use the same approach here for the message (as you've done for the id): `...to eq peep.message`

And the same with the date as well. This is because you decided to store the first test case in a variable and this is what would make more sense as per your approach.